pkg/proc/gdbserial/gdbserver.go

```diff
@@ -380,7 +388,7 @@ func getLdEnvVars() []string {
 // LLDBLaunch starts an instance of lldb-server and connects to it, asking
 // it to launch the specified target program with the specified arguments
 // (cmd) on the specified directory wd.
-func LLDBLaunch(cmd []string, wd string) (*Process, error) {
+func LLDBLaunch(cmd []string, wd string, foreground bool) (*Process, error) {
```

Comment:
You mention not being able figure out LLDB but seem to have implemented it here. If for whatever reason it truly is not supported we should return an error here and mention in the docs.

Reply:
LLDBLaunch launches either lldb or debugserver depending on the one it finds.
Returning an error would prevent connecting to lldb-server in headless mode, which still works fine except for the fact that it won't get stdin.